### PR TITLE
Add lxterminal

### PR DESCRIPTION
**Pull request by planet36**
This branch adds a template for LXTerminal.

https://github.com/lxde/lxterminal

### CODE REVIEW

File: templates/lxterminal.dot [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[general]
+bgcolor=rgb({{=c.background}})
+color_preset=Custom
```
Comment (MindTooth) [inline templates/lxterminal.dot:3]: Could this be called Srcery, or is this related to some inheritence?

Comment (planet36) [reply templates/lxterminal.dot:3]: Yes, I'll change that.
But in the preferences GUI it always says "Custom" if a non-built-in theme is used.

Comment (MindTooth) [reply templates/lxterminal.dot]: I see.  But just for concistany. 😉 

Need to either rerun or change output file as well.

**Comment by MindTooth** [comment]
Thanks!

Fix the latest nit, and I’ll merge it. 😊
